protected/humhub/modules/admin/controllers/GroupController.php

```diff
@@ -94,6 +95,21 @@ public function actionEdit()
                     '/admin/group/manage-group-users',
                     'id' => $group->id,
                 ]);
+            } else {
+                if (!empty(Yii::$app->request->post('submitReassignAll'))) {
+                    $this->view->success(Yii::t(
+                        'base',
```

Comment:
Use `AdminModule.user` here, `base` is just for very common texts as `Save` or `Edit`

Reply:
Also we should use a more meaningful text e.g:

`$this->view->info(Yii::t('AdminModule.user', 'Default space reassignment job was queued.))`

Reply:
Done!